pom.xml

```diff
@@ -44,6 +44,25 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>jakarta.validation</groupId>
```

Comment:
Why do you use this dependency ? Is not enough only to use ```spring-boot-starter-validation``` dependency for using validations from ```jakarta``` ?